benchmark_config:
  parameters:
    name: Collision Check
    runs: 5

  robot_name: panda

  collision_detectors:
    - FCL
    - Bullet

  robot_states:
    - name: ready
      robot_state:
        joint_state:
          header:
            frame_id: panda_link0
          name: [panda_joint1, panda_joint2, panda_joint3, panda_joint4, panda_joint5, panda_joint6, panda_joint7]
          position: [0, -0.785, 0, -2.356, 0, 1.571, 0.785]

  collision_requests:
    - name: requests
      distance: False
      cost: False
      contacts: True
      max_contacts: 99
      max_contacts_per_pair: 10
      max_cost_sources: 1
      verbose: False
      
  clutter_worlds:
    - name: empty-world
      n_objects: 0
      
    - name: clutter-world
      robot_state: ready
      n_objects: 1000
      object_type: MESH
      resource: package://moveit_resources_panda_description/meshes/collision/link5.stl
      scale:
        - bound: [0.3, 1.0]
